lib/croma/new1_existence.ex

```diff
@@ -0,0 +1,117 @@
+defmodule Croma.New1Existence do
+  @moduledoc false
+
+  defmodule AssumedModuleStore do
+    @moduledoc false
+
+    @spec inserted_new(module) :: :ok
+    def inserted_new(mod) do
```

Comment:
Not sure why this function is called `inserted_new`. Could you clarify? I think simply `insert` or `put` would be fine.